public/app/features/canvas/elements/droneSide.tsx

```diff
@@ -29,7 +29,7 @@ const DroneSideDisplay: FC<CanvasElementProps<DroneSideConfig, DroneSideData>> =
       xmlns="http://www.w3.org/2000/svg"
       xmlnsXlink="http://www.w3.org/1999/xlink"
       viewBox="0 0 1300 290"
-      style={{ transform: droneSidePitchTransformStyle }}
+      style={{ transform: droneSidePitchTransformStyle, fill: defaultColor }}
     >
       <g className="arms" stroke="black" strokeWidth="28px">
```

Comment:
I think we need to change the `stroke` color here to our default so we don't have the black drone on dark background (for both side + front svgs)

Also we might want to consider similar "custom" sizing for drone side / front to better match underlying svg

![Screen Shot 2022-07-29 at 12 22 40 PM](https://user-images.githubusercontent.com/22381771/181830168-6c86d416-51f1-41dd-ace1-332246086a0a.png)